tests/index/test_index_dynamic_search_rules_meilisearch.py

```
@@ -0,0 +1,99 @@
+    test_index.wait_for_task(create_response.task_uid)
+    
+    # Delete it
+    delete_response = test_index.delete_dynamic_search_rule(rule_uid)
+    assert delete_response.task_uid is not None
+    
+    test_index.wait_for_task(delete_response.task_uid)
+    
+    # Verify it's deleted - should raise error or return empty
+    try:
+        test_index.get_dynamic_search_rule(rule_uid)
+        assert False, "Rule should have been deleted"
```

Ruff (0.15.9)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)